i18n/en.i18n.json

```diff
@@ -77,6 +79,8 @@
     "label-create": "Create a new label",
     "label-delete-pop": "There is no undo. This will remove this label from all cards and destroy its history.",
     "label-default": "%s label (default)",
+    "attachment": "Attachment",
+    "attachments": "Attachments",
```

Comment:
The key should be named `attachment_plural`. See http://i18next.com/pages/doc_features.html#plurals